peer/brontide.go

```diff
@@ -3895,3 +3914,101 @@ func (p *Brontide) sendLinkUpdateMsg(msg lnwire.Message) {
 	// continue processing message.
 	chanStream.AddMsg(msg)
 }
+
+// handleLinkUpdateMsg returns a boolean to indicate whether the given message
+// can be sent to the channel's message stream. If this message's channel is
+// still pending, it will be cached.
+func (p *Brontide) handleLinkUpdateMsg(msg lnwire.Message,
+	chanID lnwire.ChannelID) bool {
+
+	p.activeChanMtx.RLock()
+	defer p.activeChanMtx.RUnlock()
+
+	channel, ok := p.activeChannels[chanID]
```

Comment:
Can we reuse `isActiveChannel` and `isPendingChannel` for clarity here?

Reply:
wanted this too but couldn't as they were in two separate locks, which complicated things. but now it's changed to sync maps, so yeah switched.